pkg/apis/scheduling/v1alpha1/types.go

```diff
@@ -182,6 +182,8 @@ type Queue struct {
 // QueueSpec represents the template of Queue.
 type QueueSpec struct {
 	Weight int32 `json:"weight,omitempty" protobuf:"bytes,1,opt,name=weight"`
+	// MaxPodGroupNumber defines the max number of running podGroup in one queue, default to infinite.
+	MaxPodGroupNumber int32 `json:"maxPodGroupNumber,omitempty" protobuf:"bytes,1,opt,name=maxPodGroupNumber"`
```

Comment:
`Capability` is better :)

Reply:
Is `Capability` just for limiting pod group number? Then its type is `int32`. Or  will `Capability` contain other resources, e.g. `CPU`, `Mem`?

Reply:
also contain other resources for extensibility, e.g. v1.Resources.